lib/hanami/commands/generate/action.rb

```diff
@@ -234,14 +234,16 @@ def spec_root
         end
 
         def relative_action_path
-          path = Pathname.new('.').join('..', '..', '..')
-          path = path.join('..') if environment.container?
+          nestings = ['..'] * @controller_pathname.count('/')
```

Comment:
@jodosha I've looked into it, and I think using `File::SEPARATOR` would break functionality on Windows here.

At the top of `action.rb`, the `@controller_pathname` variable is assigned by using `Hanami::Utils::String().underscore`.

`#underscore` will only replace class divisions with `/` ([see here](https://github.com/hanami/utils/blob/master/lib/hanami/utils/string.rb#L43)).

If we wanted to use `File::SEPARATOR` here we'd have to adjust `#underscore` to use it as well.

It would work for `hanami g action foo\bar#baz` but it would break for `hanami g action Foo::Bar::Baz` - also, as mentioned below, it won't work for `foo\bar\baz`